pwiz_tools/Skyline/Model/Lib/EncylopeDiaLibrary.cs

```diff
@@ -205,7 +205,7 @@ private bool Load(ILoadMonitor loader)
         // ReSharper disable LocalizableElement
         private bool LoadLibraryFromDatabase(ILoadMonitor loader)
         {
-            var status = new ProgressStatus(
+            IProgressStatus status = new ProgressStatus(

```

Comment:
Just curious - why not var?

Reply:
Because you need IProgressStatus and not ProgressStatus. Pretty common in our code.

Reply:
Huh, I would have thought that amounted to the same thing.

Reply:
ReSharper knows better. Give it a try. It has to do with the use of "ref" in the function call below:

SetLibraryEntries(FilterInvalidLibraryEntries(ref status, spectrumInfos));